impeller/entity/contents/vertices_contents.cc

```diff
@@ -15,15 +17,18 @@
 
 namespace impeller {
 
-VerticesContents::VerticesContents(Vertices vertices)
-    : vertices_(std::move(vertices)){};
+VerticesContents::VerticesContents() = default;
 
 VerticesContents::~VerticesContents() = default;
 
 std::optional<Rect> VerticesContents::GetCoverage(const Entity& entity) const {
-  return vertices_.GetTransformedBoundingBox(entity.GetTransformation());
+  return geometry_->GetCoverage(entity.GetTransformation());
 };
 
+void VerticesContents::SetGeometry(std::unique_ptr<Geometry> geometry) {
+  geometry_ = std::move(geometry);
+}
+
```

Comment:
Passing this into VerticesContents to start using it right away is a cool idea.

Reply:
This and the GetCoverage seem like something that could eventually just be members of the base contents class, or perhaps of some base GeometryContents class?

Reply:
This should go on the `ColorSourceContents` base class once we start upgrading the color sources. Some `Contents` (namely `FilterContents`) take other `Contents` as input as opposed to `Geometry`.

Reply:
(`VerticesContents` should also go away and/or be renamed to [`ColorColorSourceContents`](https://flutter.dev/go/impeller-geometry#heading=h.shwijaa6a6hf) in the end)